### DISCUSSION TITLE
One dependency update email for two repos

### DISCUSSION THREAD

**Discussion by eygraber** [renovatebot/renovate#31933] [Request Help] [answered]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

I use renovatebot with my repos and they all get a PR email when there's a dependency update. However, two of my repos get a single email when there's an update for a dependency they both have. This frequently causes me to miss updating the dependency on the second repo (because I usually just click the link for the first one).

The repos are

https://github.com/eygraber/ejson-kotlin
https://github.com/eygraber/uri-kmp

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment, marked as answer]
This is GitHub sending emails, not Renovate or Mend. Nothing we can help you with.

**Reply by eygraber** [reply]
This only happens with renovate PRs though and only for these two repos.
